tests/vm

```diff
@@ -34,6 +43,12 @@ if ! echo "${LXD_SNAP_CHANNEL}" | grep -qE '^4\.0/'; then
     lxc exec vm1 -- sleep 60 || exitCode=$?
     [ "${exitCode:-0}" -eq 129 ]
     wait $!
+
+    echo "==> Test what happens when the connection with the agent is severed"
+    # Simulates what happens during lxc stop
+    lxc exec vm1 -- killall -s SIGTERM lxd-agent || exitCode=$?
+    [ "${exitCode:-0}" -eq 129 ]
+    wait $!
```

Comment:
I don't think there is any background job to wait for at this point, is there?

Reply:
Whoops, that slipped me, thanks for catching